packages/filebrowser-extension/tsconfig.json

```diff
@@ -18,9 +18,6 @@
     {
       "path": "../docmanager"
     },
-    {
-      "path": "../docregistry"
-    },
```

Comment:
did this sneak in accidentally?

Reply:
This is part of the dependency fix above.